internal/ent/schema/mappedcontrol.go

```diff
@@ -0,0 +1,108 @@
+package schema
+
+import (
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+
+	"github.com/theopenlane/core/internal/ent/generated/privacy"
+	"github.com/theopenlane/core/internal/ent/mixin"
+	"github.com/theopenlane/core/internal/ent/privacy/policy"
+	emixin "github.com/theopenlane/entx/mixin"
+)
+
+// MappedControl holds the schema definition for the MappedControl entity
+type MappedControl struct {
+	ent.Schema
+}
+
+// Fields of the MappedControl
+func (MappedControl) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("control_id").
+			Comment("the id of the control being mapped").
+			Unique().
+			Immutable().
```

Comment:
we would not want to make this unique or immutable I think; we already assign an `ID` field to the table for uniqueness, and control ID's can change from the standards themselves (or they are collapsed together, or 2 existing ones in a new revision of a standard may get entirely new codes in the one they are collapsed into). Either that, or we need to be sure we can easily / elegantly retain versions of the controls and then create a new standard with a new revision and new control sets, it will just be confusing to the client if year over year they're expecting their controls from the prior period to be re-used when creating a new program / standard